optimizely/decision/composite_feature_service.go

```diff
@@ -28,14 +28,14 @@ var cfLogger = logging.GetLogger("CompositeFeatureService")
 
 // CompositeFeatureService is the default out-of-the-box feature decision service
 type CompositeFeatureService struct {
-	experimentDecisionService ExperimentService
+	featureExperimentDecisionService ExperimentService
```

Comment:
nit: We don't need the word `Decision` as it is redundant. So `featureExperimentService`